### PR TITLE
Migrate CompatibilityTable to Compat macro (es)

### LINKED ISSUE #354: Migrate CompatibilityTable to Compat macro
I have created a macro that migrates `{{CompatibilityTable}}` macro to `{{Compat()}}` macro.
https://gist.github.com/YujiSoftware/420641a79d12c074a96210e644953038

First, apply to Japanese.
Apply to other languages if needed.

----

`{{CompatibilityTable}}` マクロを `{{Compat()}}` マクロに移行するマクロを作成しました。
これを、MDN に適用します。

まずは、日本語に適用します。
必要であれば、他の言語にも適用します。

### PR DESCRIPTION
**Pull request by YujiSoftware**
# Related Issue

fix #354

# Work

I applied the issue-described site compatibility table migration script to the Compat macro for the documentation.
This script does the following:

* Delete {{CompatibilityTable}} to the line before the next h2 tag in each file
* Added the {{Compat()}} macro described in the English version (of the same path)

### CODE REVIEW

**Comment by cw118** [comment]
@YujiSoftware Thanks so much for making these PRs for the locales that need it :)

**Just a note to the l10n-es review team:** many of the Compat macros here could/should have no argument and instead have a `browser-compat` line in the YAML front-matter, but either way will work just fine. I'll leave it to you all to decide how to proceed with this PR and whether you'd like to move to front-matter all in one or in another pull request.

File: files/es/games/techniques/webrtc_data_channels/index.html [hunk 1]
```diff
@@ -27,59 +27,8 @@ <h2 id="Usando_la_libreria_p2p">Usando la libreria p2p</h2>
 <p><strong>Nota:</strong> Continuaremos agregando contenido aquí pronto; hay algunos problemas de organización por resolver.</p>
 </div>
 
-<h2 id="Compatibilidad_de_navegadores">Compatibilidad de navegadores</h2>
 
-<p>{{ CompatibilityTable() }}</p>
```
Comment (davbrito) [inline files/es/games/techniques/webrtc_data_channels/index.html:31]: Here the script didn't add the replacement with the `{{Compat}}` macro. Is this intended?

**Review by davbrito — CHANGES_REQUESTED** [review]

**Review by JuanVqz — APPROVED** [review]
it's hard to review all the pages but until I can see it's working as expected and doing what it says that will do

File: files/es/games/techniques/webrtc_data_channels/index.html (hunk 1 above)
Comment (davbrito) [reply files/es/games/techniques/webrtc_data_channels/index.html:31]: Cool. Thanks @cw118!

**Review by davbrito — APPROVED** [review]

**Comment by JuanVqz** [comment]
@YujiSoftware thank you for your contribution we appreciate it :100:

**Pull request merged**